pyaedt/rmxprt.py

```diff
@@ -247,3 +247,60 @@ def design_type(self):
     @pyaedt_function_handler()
     def design_type(self, value):
         self.design_solutions.design_type = value
+
+    @pyaedt_function_handler()
+    def create_setup(self, setupname="MySetupAuto", setuptype=None, **kwargs):
+        """Create a new analysis setup for RmXport.
+        Optional arguments are passed along with ``setuptype`` and ``setupname``.  Keyword
+        names correspond to the ``setuptype``
+        corresponding to the native AEDT API.  The list of
+        keyword here is not exhaustive.
+
+
+        Parameters
+        ----------
+        setuptype : int, str, optional
+            Type of setup. Must be one of the following:
+            "IcepakSteadyState", "IcepakTransient".
+            Default: "IcepakSteadyState"
+        setupname : str, optional
+            Name of the setup. Default: "Setup1"
+        **kwargs : dict, optional
+            Available keys depend on setup chosen:
+
+            :doc:`../SetupTemplatesRmxprt`.
```

Comment:
```suggestion
        """Create an analysis setup for RmXport.
        
        Optional arguments are passed along with the ``setuptype`` and ``setupname``
        parameters. Keyword names correspond to the ``setuptype``
        corresponding to the native AEDT API.  The list of
        keywords here is not exhaustive.

        Parameters
        ----------
        setuptype : int, str, optional
            Type of the setup. Options are "IcepakSteadyState"
            and "IcepakTransient". The default is "IcepakSteadyState".
        setupname : str, optional
            Name of the setup. The default is "Setup1".
        **kwargs : dict, optional
            Available keys depend on the setup chosen.
            For more information, see :doc:`../SetupTemplatesRmxprt`.
```
Here it's RmXport. It should either be RMxport or RMXport. Please check.

Reply:
The AEDT Product name is RMXprt